### ISSUE TITLE
FTS$UUID - more than 16 characters

### ISSUE THREAD

**Issue by vanicekv** [IBSurgeon/lucene_udr#12]
Hello,

We have a database in which the primary keys are as CHAR(18), using triggers we fill the additional FTS$UUID with 16 characters from that PK (substring). Most of the time this works without any problem, but for large databases with a lot of data there will be collision, is it possible to extend the FTS$UUID to 18 characters?

Thank you.

-----

FTS$UUID - value of a key field of type BINARY(16). This type is used to store the GUID;

**Comment by sim1984** [comment]
No. Let me remind you that Firebird does not have a separate GUID type. And the library cannot build full-text indexes with any arbitrary primary key. The most frequently used forms of the primary key are embedded in it.
